core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -998,6 +998,23 @@ public long getMajorCompactionSize() {
     return compactionSize;
   }
 
+  /**
+   * returns minor compaction size value from carbon properties or 0 if it is not valid or
+   * not configured
+   *
+   * @return compactionSize
+   */
+  public long getMinorCompactionSize() {
+    long compactionSize = 0;
+    try {
+      compactionSize = Long.parseLong(getProperty(
```

Comment:
please handle if user sets negative value.
user can set from 0 to LONG_MAX I think.

Reply:
@ajantha-bhat handled